src/stores/app/livekit-store.ts

```
@@ -243,6 +245,53 @@ export const useLiveKitStore = create<LiveKitState>((set, get) => ({
+              message: 'Failed to attach audio track',
+              context: { error: err, trackSid: track.sid },
+            });
+          }
+        }
+      });
+
+      room.on(RoomEvent.TrackUnsubscribed, (track, publication, participant) => {
+        // On web, detach and remove audio elements
+        if (Platform.OS === 'web' && track.kind === Track.Kind.Audio) {
+          try {
+            track.detach().forEach((el) => el.remove());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)